test/components/general/button.test.js

```diff
@@ -12,4 +12,15 @@ describe("Button Component", () => {
     const wrapper = shallow(<Button type="big">Button</Button>);
     expect(wrapper.props()).toEqual({ children: "Button", type: "big" });
   });
+
+  it("sets onClick to the value of the clickHandler prop", () => {
```

Comment:
Nitpick: I don't think this is really the thing you're testing here—would a spec title like "calls the click handler when clicked" be better?